Kickstarter-iOS/Views/Cells/NoRewardCell.swift

```diff
@@ -12,9 +12,7 @@ internal final class NoRewardCell: UITableViewCell, ValueCell {
   @IBOutlet fileprivate weak var rootStackView: UIStackView!
   @IBOutlet fileprivate weak var copyStackView: UIStackView!
 
-  internal func configureWith(value project: Project) {
-    self.contentView.backgroundColor = Library.backgroundColor(forCategoryId: project.category.rootId)
-  }
+  internal func configureWith(value: Void) {}
```

Comment:
Might be worth revisiting naming here:

``` diff
-configureWith(value:)
+configure(with:)
```

That way, we could define the function with a default:

```diff
-configureWith(value: Void) {}
+configure(with value: Void = ()) {}
```

And call it as such:

```diff
-cell.configureWith(value: ())
+cell.configure()
```